MAINTAINERS.yml

```diff
@@ -2346,18 +2346,26 @@ Nuvoton NPCX Platforms:
   labels:
     - "platform: Nuvoton NPCX"
 
-Nuvoton Numicro Platforms:
-  status: odd fixes
+Nuvoton Numicro Numaker Platforms:
+  status: maintained
+  maintainers:
+    - cyliangtw
   collaborators:
     - ssekar15
+    - cyliangtw
```

Comment:
Please remove yourself from the collaborators list, since you are already the maintainer